src/main/java/gui/utility/EventHelper.java

```diff
@@ -14,20 +15,20 @@
  */
 public class EventHelper {
 
-    public static Entry<String> eventReaderToEntry(EventReader eventReader) {
-        Entry<String> entry = new Entry<>(eventReader.getName());
-        entry.changeStartTime(eventReader.getStartTime());
-        entry.changeEndTime(eventReader.getEndTime());
+    public static Entry<String> eventInfoToEntry(EventInfo eventInfo) {
+        Entry<String> entry = new Entry<>(eventInfo.getName());
+        entry.changeStartTime(eventInfo.getStartTime());
+        entry.changeEndTime(eventInfo.getEndTime());
         LocalDate date = LocalDate.of(2021, 12, 8);
```

Comment:
This might be a forgotten constant. I'm guessing it should be injected into this method instead?

Reply:
right it was for me to test the view. I will include the change in another pr with the calendar updates and creation.